File: 1066.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var par = 0;var impar = 0;var positivo = 0;var negativo = 0;
for(var c = 1; c < 6; c++){
    var ca = Number(lines.shift());
    if(ca%2 === 0){
        var par = par + 1}
    else if(ca&2 !== 0){
        var impar = impar + 1
    }
    if(ca > 0){
        var positivo = positivo + 1
    }
    else if(ca < 0){
        var negativo = negativo + 1
    }}
console.log(`${par} valor(es) par(es)
${impar} valor(es) impar(es)
${positivo} valor(es) positivo(s)
${negativo} valor(es) negativo(s)`);

File: 1066.py
par = impar = positivo = negativo = 0
for c in range(1,6):
    ca = int(input())
    if ca%2 == 0:
        par += 1
    elif ca&2 != 0:
        impar += 1
    if ca > 0:
        positivo += 1
    elif ca < 0:
        negativo += 1
print(f'''{par} valor(es) par(es)
{impar} valor(es) impar(es)
{positivo} valor(es) positivo(s)
{negativo} valor(es) negativo(s)''')
